svm/src/transaction_processor.rs

```diff
@@ -344,15 +347,17 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
             sanitized_txs.len(),
         );
 
-        timings.saturating_add_in_place(
+        execute_timings.saturating_add_in_place(
```

Comment:
Do we still need to call `saturating_add_in_place()` since `execute_timings` is on the stack and it is not accumulating over multiple calls to the function?

Reply:
I thought this was just the preferred method for updating on `&mut self`, rather than doing something like:

```rust
execute_timings.metrics.program_cache_time = execute_timings.metrics.program_cache_time
    .saturating_add(program_cache_time.as_us());
```

Reply:
Would we instead use `saturating_add_assign`?

Reply:
Simple assignment should have worked, but looks like `ExecuteTimings` doesn't have any such methods. Maybe it's ok to leave it as is.

Reply:
Up to you. Lmk if you want it adjusted.